sass/components/_mentions.scss

```diff
@@ -26,11 +26,16 @@
 
 .mention--highlight {
     padding: 0 1px;
-    background-color: $yellow;
+    background-color: var(--mention-highlight-bg);
     border-radius: 4px;
 
+    .mention-link {
+        color: var(--mention-highlight-link);
```

Comment:
This colour doesn't seem right. See screenshot of sysadmin mention below 
![Screen Shot 2022-11-21 at 10 02 42 AM](https://user-images.githubusercontent.com/14115924/203087801-dd23be86-6398-4f5b-84fb-3e2080ac08a1.png)

Reply:
Ah you are correct, I got mixed up with the theming